src/pages/VisualizationPage/index.tsx

```diff
@@ -100,46 +105,57 @@ const VisualizationPage = () => {
 
   return (
     <Wrapper>
-      <div className="upload-link">
+      <PageSection variant="light">
+        <InfoIcon
+          title="DICOM Viewer"
+          p1={
+            <Paragraph style={infoStyle}>
+              Here you get to upload all your files.
```

Comment:
![screenshot](https://user-images.githubusercontent.com/20404439/195736273-aeac8de4-afbd-4e84-995a-713aaa167cf4.png)

Please update the tooltip. @PintoGideon @rudolphpienaar what do you think we should say here? Maybe...

Reply:
I have updated it with the current suggestion.